cluster/config.go

```diff
@@ -109,6 +109,11 @@ func BuildContainerConfig(c dockerclient.ContainerConfig) *ContainerConfig {
 	// remove affinities/constraints/reschedule policies from env
 	c.Env = env
 
+	if parts := strings.SplitN(c.HostConfig.NetworkMode, "/", 2); len(parts) == 2 {
```

Comment:
thanks, I'm going to change to [1:]. regarding the `/` in name, I don't think it can be supported in swarm right now